sklearn/feature_selection/univariate_selection.py

```diff
@@ -47,8 +47,10 @@ def f_oneway(*args):
 
     Parameters
     ----------
-    sample1, sample2, ... : array_like, sparse matrices
-        The sample measurements should be given as arguments.
+    *args : array_like, sparse matrices
```

Comment:
I think I prefer the former, but making the script happy may be fiddly (which is why I didn't initially suggest it for a test).